process/peer/process_test.go

```diff
@@ -54,6 +54,14 @@ func createMockPubkeyConverter() *mock.PubkeyConverterMock {
 	return mock.NewPubkeyConverterMock(32)
 }
 
+func createCommonHeaderCacheMap(cacheMap map[string]data.HeaderHandler) map[string]data.CommonHeaderHandler {
```

Comment:
This is duplicated with the one in export_test.go . You can make it there exported -> CreateCommonHeaderCacheMap and remove this one

Reply:
they are in completely different folders - it is a no go.

Reply:
Sebi is right, they are the same. Please remove this one and export the one from export_test.go file

Reply:
alright. done.